pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -573,6 +573,9 @@ with stdenv.lib;
   ''}
   MEDIA_USB_SUPPORT y
   MEDIA_PCI_SUPPORT y
+  MEDIA_ANALOG_TV_SUPPORT y
+  VIDEO_STK1160_COMMON m
+  VIDEO_STK1160_AC97 y
```

Comment:
Which kernel version has `VIDEO_STK1160_AC97`? I only see:
````
drivers/media/usb/Makefile:obj-$(CONFIG_VIDEO_STK1160) += stk1160/
drivers/media/usb/stk1160/Kconfig:config VIDEO_STK1160_COMMON
drivers/media/usb/stk1160/Kconfig:config VIDEO_STK1160
drivers/media/usb/stk1160/Kconfig:      depends on VIDEO_STK1160_COMMON
drivers/media/usb/stk1160/Makefile:obj-$(CONFIG_VIDEO_STK1160) += stk1160.o
````

Reply:
4.9 kernel still has it (which i am using right now). It seems to have been removed with https://patchwork.linuxtv.org/patch/37808/ . according to https://cateee.net/lkddb/web-lkddb/VIDEO_STK1160_AC97.html it is available in "Linux kernels: 3.7–3.19, 4.0–4.10, 4.10+HEAD"